app/controllers/system_notifications_controller.rb

```diff
@@ -26,6 +30,13 @@ def mark_as_seen
     render json: { result: 'failed' }
   end
 
+  def mark_as_read
```

Comment:
`mark_as_read` and `mark_as_seen` could be moved to `service` object. 
If it stays as it is, maybe `mark_as_read` could be changed from class to an instance method and you don't need to passing ID.

Reply:
Main idea, that we can pass multiple id’s and update seen_at with one SQL query.

Reply:
@radiokills your opinion here?

Reply:
This model currently small and this logic we can only use in this model, i think we need move it to service if we will use it in another model. Also i think leave it like class method and give ability to receive multiple id's, cause someone in future may want "Mark All As Read" button.